go/interpreter/lfvm/ct/ct.go

```diff
@@ -17,12 +17,17 @@ import (
 	"github.com/Fantom-foundation/Tosca/go/ct/common"
 	"github.com/Fantom-foundation/Tosca/go/ct/st"
 	"github.com/Fantom-foundation/Tosca/go/ct/utils"
+	"github.com/Fantom-foundation/Tosca/go/interpreter/lfvm"
 	"github.com/Fantom-foundation/Tosca/go/tosca"
 	lru "github.com/hashicorp/golang-lru/v2"
+	"github.com/holiman/uint256"
 )
```

Comment:
Just looking at the import list of this module, it makes me think that this code relates more to CT than to lfvm. I think it may be better there.

Reply:
actually, if you look into the other folder of `go/interpreters/` this is the correct place for the `ct` folder, it is perhaps not the right place for the lfvm implementaiton. But the idea of the `interpreter` folder was to remove the `ct` adaptors from the ct implementation packages. Perhaps `lfvm` package should be moved to another folder. @HerbertJordan